rundeckapp/grails-app/domain/rundeck/ScheduledExecutionStats.groovy

```diff
@@ -17,6 +19,15 @@ class ScheduledExecutionStats {
         content type: 'text'
     }
 
+    static ScheduledExecutionStats getOrCreate(String jobUuid) {
```

Comment:
This class doesn't need this method. The one in the provider should be preferred.